ipaserver/install/server/upgrade.py

```diff
@@ -529,6 +529,37 @@ def named_remove_deprecated_options():
     return True
 
 
+def named_add_ipa_ext_conf_include():
+    """
+    Ensures named.conf does include the ipa-ext.conf file
+    """
+    if not bindinstance.named_conf_exists():
+        logger.info('DNS is not configured')
+        return False
+
+    if not bindinstance.named_conf_include_exists(paths.NAMED_CUSTOM_CONFIG):
+        bindinstance.named_conf_add_include(paths.NAMED_CUSTOM_CONFIG)
+        return True
+    return False
+
+
+def named_add_ipa_ext_conf_file():
```

Comment:
This function looks pretty much the same as its counterpart in ipaserver/install/bindinstance.py.
You could move it out to ipaserver/install/bindinstance.py.

Reply:
and call it from within "upgrade" in order to ensure the file get added? I'm new to this project, trying to figure out how things are working together :).